src/lib.rs

```diff
@@ -76,24 +76,21 @@ impl CmdLn {
 impl Drop for CmdLn {
     fn drop(&mut self) {
         if !self.raw.is_null() {
-            let ref_count = unsafe { bindings::cmd_ln_free_r(self.raw) };
-            assert!(ref_count == 0);
```

Comment:
I had to remove the assertion because ps_init() increments the refcount multiple times.